common/src/main/scala/org/apache/celeborn/common/metrics/MetricsSystem.scala

```diff
@@ -81,10 +81,10 @@ class MetricsSystem(
   }
 
   def getSourcesByName(sourceName: String): Seq[Source] =
-    sources.filter(_.sourceName == sourceName).toSeq
+    sources.asScala.filter(_.sourceName == sourceName).toSeq
```

Comment:
nit: We can drop the `toSeq`

Reply:
updated.

Reply:
reverted, as the `toSeq` is required for Scala 2.13.

```
[error] /home/runner/work/incubator-celeborn/incubator-celeborn/common/src/main/scala/org/apache/celeborn/common/metrics/MetricsSystem.scala:84:27: type mismatch;
[error]  found   : scala.collection.mutable.Buffer[org.apache.celeborn.common.metrics.source.Source]
[error]  required: Seq[org.apache.celeborn.common.metrics.source.Source]
[error]     sources.asScala.filter(_.sourceName == sourceName)
[error]                           ^
[error] one error found
```